opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -582,7 +580,14 @@ public void onPause() {
     private class RefreshGeowidgetReceiver extends BroadcastReceiver {
         @Override
         public void onReceive(Context context, Intent intent) {
-            listTaskPresenter.refreshStructures();
+            Bundle extras = intent.getExtras();
+            if (extras != null && extras.getBoolean(UPDATE_LOCATION_BUFFER_RADIUS)) {
+                String bufferRadiusStr = RevealApplication.getInstance().getGlobalConfigs().get(LOCATION_BUFFER_RADIUS_IN_METRES);
+                Float bufferRadius = bufferRadiusStr == null ? DEFAULT_LOCATION_BUFFER_RADIUS_IN_METRES : Float.valueOf(bufferRadiusStr);
+                kujakuMapView.setLocationBufferRadius(bufferRadius);
```

Comment:
Make a common function in Utils so that line 585-587 is not repeated in the GeowidgetFactory

Reply:
Done.